### PR TITLE
added az.compare

### PR DESCRIPTION
**Pull request by nsiccha**
## Description

Replaced modelwise elpd computation by az.compare.

## Checklist

- [x] Tests covering the new functionality have been added
- [x] Documentation has been updated OR the change is too minor to be documented
- [x] Changes are listed in the `CHANGELOG.md` OR changes are insignificant

### CODE REVIEW

File: notebooks/01-ym-prototype-workflow.ipynb [hunk 1]
```diff
@@ -22,7 +22,8 @@
      "name": "stderr",
```
Comment (nsiccha) [inline notebooks/01-ym-prototype-workflow.ipynb:22]: <p>I'm am famous now.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/yannmclatchie/kulprit/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

**Review by yannmclatchie — APPROVED** [review]
Looks good, cheers!

File: notebooks/01-ym-prototype-workflow.ipynb (hunk 1 above)
Comment (aloctavodia) [inline notebooks/01-ym-prototype-workflow.ipynb:22]: <p>not directly related to this PR, but while you are here could&nbsp;you remove the line   <code>plt.show()</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/yannmclatchie/kulprit/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Comment (aloctavodia) [inline notebooks/01-ym-prototype-workflow.ipynb:22]: <p>idem</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/yannmclatchie/kulprit/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

**Comment by aloctavodia** [comment]
oops, I am late to the party.

File: notebooks/01-ym-prototype-workflow.ipynb (hunk 1 above)
Comment (yannmclatchie) [reply notebooks/01-ym-prototype-workflow.ipynb:22]: <p>I'll update this in the newest prototype workflow for the next PR</p>

**Pull request merged**